tenseal/sealapi/sealapi_util_namespace.cpp

```diff
@@ -159,33 +160,42 @@ void bind_util_namespace(pybind11::module &m) {
              [](const RNSTool &obj, std::uint64_t *input, std::size_t count,
                 const NTTTables *rns_ntt_tables) {
                  obj.divide_and_round_q_last_ntt_inplace(
-                     RNSIter(input, count), rns_ntt_tables, MemoryManager::GetPool());
+                     RNSIter(input, count), rns_ntt_tables,
+                     MemoryManager::GetPool());
              })
         .def("fastbconv_sk",
-             [](const RNSTool &obj, const std::uint64_t *input, std::size_t count,
-                std::uint64_t *destination) {
-                 obj.fastbconv_sk(ConstRNSIter(input, count), RNSIter(destination, count), MemoryManager::GetPool());
+             [](const RNSTool &obj, const std::uint64_t *input,
+                std::size_t count, std::uint64_t *destination) {
+                 obj.fastbconv_sk(ConstRNSIter(input, count),
+                                  RNSIter(destination, count),
+                                  MemoryManager::GetPool());
              })
         .def("sm_mrq",
-             [](const RNSTool &obj, const std::uint64_t *input, std::size_t count,
-                std::uint64_t *destination) {
-                 obj.sm_mrq(ConstRNSIter(input, count), RNSIter(destination, count), MemoryManager::GetPool());
+             [](const RNSTool &obj, const std::uint64_t *input,
```

Comment:
I think pybind doesn't map uint64_t* to a vector, it will consider it a single value.

I think it better to use vector instead of (uint64, size_t). and call it using (input.data(), input.size()).


LE: Now I see that the issue was there all along, it's not new. It was documented here https://github.com/OpenMined/TenSEAL/blob/master/tenseal/sealapi/sealapi_util_namespace.cpp#L30

but we didn't fix it everywhere.

Please replace the (std::uint64_t *, size_t) parameters with std::vector, to have the full functionality

Reply:
Thans for review. I updated the codes to use std::vector as params, also since the seal's `ConstRNSIter` and `RNSIter` need a poly_modulus_degree param which is not equal with vector size, so add `poly_modulus_degree` in some places.